pymitsubishi/mitsubishi_parser.py

```diff
@@ -635,6 +680,45 @@ def convert_temperature_to_segment(temperature: int) -> str:
     return format(int(value), "02x")
 
 
+def celsius_to_enhanced_temperature(temperature: float) -> bytes:
+    """Convert temperature (Celsius) to enhanced temperature, ref https://muart-group.github.io/developer/it-protocol/data-types/temperature-units#enhanced-temperatures"""
+    value = int((temperature * 2) + 128)
```

Comment:
We should probably use round instead of int here right? Cause if my external thermostat reports 12.9 this would be reported as 153,8 -> 153 -> 12.5 and not as the more sensible 13 degree value.